src/main/java/ch/njol/skript/expressions/ExprRawString.java

```diff
@@ -0,0 +1,94 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionList;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.VariableString;
+import ch.njol.skript.util.LiteralUtils;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+
+@Name("Raw String")
+@Description("Returns the string without formatting (colors etc.) and without stripping them from it, " +
+	"e.g. <code>raw \"&aHello There!\"</code> would output <code>&aHello There!</code>")
+@Examples("send raw \"&aThis text is unformatted!\" to all players")
+@Since("INSERT VERSION")
+public class ExprRawString extends PropertyExpression<String, String> {
```

Comment:
This should be a SimpleExpression, you only have one syntax, and you're registering it as a Simple expression already in the register static block below.

I'm assuming your reference was ExprColoured, and honestly I have no clue why that class is a PropertyExpression either. It probably started out as one, and got scratched. It's a very old class and isn't up to date.

Reply:
When I first made it, it was actually a simple expression but ayham told me to make it a property expression 🤷